ci/goreleaser/goreleaser.yml

```diff
@@ -17,6 +17,8 @@ builds:
       - linux
     goarch:
       - amd64
+    env:
+      - GO111MODULE=off
```

Comment:
Rather than burying it inside the goreleaser template, this is addressed in https://github.com/TykTechnologies/gromit/blob/master/policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl#L113

Reply:
This variable should be defined in release.yml, not in goreleaser.yml.